manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: maistra/istio-ubi8-operator:2.1.0
-    createdAt: 2021-06-10T10:42:28IST
+    createdAt: 2021-07-07T12:02:04CEST
```

Comment:
Rob always ask to revert these changes - I forgot why. It doesn't hurt to me though.

Reply:
Yeah, I typically revert them, too. Forgot to do it this time, but I've done it now.